src/clj/game/cards-resources.clj

```diff
@@ -151,9 +152,9 @@
    "Eden Shard"
    {:abilities [{:effect (effect (trash card {:cause :ability-cost}) (draw :corp 2))
                  :msg "force the Corp to draw 2 cards"}]
-    :install-cost-bonus (req (if (and run (= (:server run) [:rd]) (= 0 (:position run)))
+    :install-cost-bonus (req (if (and run (= (:server run) [:rd]) (zero? (:position run)))
```

Comment:
Didn't realise the `nil` issues until after this I think. Will probably revert this change to be sure.